pre_commit/resources/hook-tmpl

```diff
@@ -153,7 +154,8 @@ def _pre_push(stdin):
                     opts = ('--origin', local_sha, '--source', source)
 
     if opts:
-        return opts
+        remote_opts = ("--push-remote-name", remote_name, "--push-remote-url", remote_url)
```

Comment:
huh weird why isn't CI running on this

Reply:
I'm not sure I got this comment.

Reply:
ah a bit of two things:

- it's slightly ironic that you didn't run `pre-commit` while contributing to `pre-commit`
- I expected azure pipelines to have triggered a build against this branch and shown github status but it didn't